pytrip/cube.py

```diff
@@ -462,16 +463,23 @@ def create_dicom_base(self):
         ds.SOPInstanceUID = '1.2.3'  # !!!!!!!!!!
         ds.StudyInstanceUID = '1.2.3'  # !!!!!!!!!!
         ds.FrameofReferenceUID = '1.2.3'  # !!!!!!!!!
-        ds.StudyDate = '19010101'  # !!!!!!!
-        ds.StudyTime = '000000'  # !!!!!!!!!!
+        ds.SeriesInstanceUID = UID.generate_uid(prefix="2.25.")
+        ds.StudyDate = datetime.datetime.today().strftime('%Y%m%d')
+        ds.StudyTime = datetime.datetime.today().strftime('%H%M%S')
         ds.PhotometricInterpretation = 'MONOCHROME2'
         ds.SamplesPerPixel = 1
         ds.ImageOrientationPatient = ['1', '0', '0', '0', '1', '0']
         ds.Rows = self.dimx
         ds.Columns = self.dimy
         ds.SliceThickness = str(self.slice_distance)
-
         ds.PixelSpacing = [self.pixel_size, self.pixel_size]
+
+        # Add eclipse friendly IDs
```

Comment:
Ok, I only tried with StudyID and I saw it failed. But maybe I missed the case or something.

Also, I was tired of guessing how vendors spell the tags. See the pinnacle case #224 

ROIContours vs ROIContourSequence   (<- no plural s)
RTROIObservations vs RTROIObservationsSequence  (<- including plural s, wtf.)

Feel free to fix what you want to fix, I cannot touch this branch anymore before next week.